submissions/AndriiZahumnyi/a-tiny-js-world-pre-oop/index.js

```diff
@@ -0,0 +1,81 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/AndriiZahumnyi/a-tiny-JS-world
+   Web app: https://andriizahumnyi.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+
+
+const man = {
+   species: 'human',
+   name: 'John',
+   gender: 'male',
+   legs: 2,
+   hands: 2,
+   saying: 'Hi! How are you?',
+   friendlyTo: ['Toby', 'Tereza', 'Tom']
+};
+
+const woman = {
+   species: 'human',
+   name: 'Tereza',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: 'I`m fine, thanks!',
+   friendlyTo: ['Tom']
+};
+
+const dog = {
+   species: 'dog',
+   name: 'Toby',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'woof-woof!',
+   friendlyTo: ['John', 'Tereza']
+};
+
+const cat = {
+   species: 'cat',
+   name: 'Tom',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'meow-meow!',
+   friendlyTo: ['Tereza']
+};
+
+const world = [man, woman, dog, cat];
+
+printArrayOfObjects = arr => {
+   arr.forEach(obj =>
+      print(obj.species + '; ' + obj.name + '; ' + obj.gender + '; ' +
+         obj.legs + '; ' + obj.hands + '; ' + obj.saying + '; ' + obj.friendlyTo));
```

Comment:
There is a pattern here. How about having a list of properties and get them joined with `Array.join`?

Reply:
Thank you for review!
I knew about this one, through telegram channel, just procrastinate.